Src/PerformanceCollector/Shared/Implementation/QuickPulse/Process/IQuickPulseTopCpuCollector.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Interface for top CPU collector.
+    /// </summary>
+    internal interface IQuickPulseTopCpuCollector
+    {
+        bool InitializationFailed { get; }
+
+        bool AccessDenied { get; }
+
+        IEnumerable<Tuple<string, int>> GetTopProcessesByCpu(int topN);
```

Comment:
I've noticed there aren't a lot of doc comments in this code, it's already quite a long code review and the lack of comments makes it even harder for me to review.